lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
1. then it makes sense drop the `-preview` for the package version as well?
2. the diff certainly is a lot more than deprecating 1 field. may be the previous changes to spec weren't generated and checked in to the sdk. given that, i don't mind you sticking with `2.0.0` versioning :)